src/QMCWaveFunctions/BasisSetFactory.cpp

```diff
@@ -232,6 +232,10 @@ BasisSetBuilder* BasisSetFactory::createBasisSet(xmlNodePtr cur,xmlNodePtr  root
     }
   }
 #endif //!QMC_COMPLEX
+  else
```

Comment:
Move this to after the OHMMS_DIM #endif so that this failure is caught in general?